test/browser/browser.sh

```diff
@@ -33,33 +26,20 @@ echo root:foobar | chpasswd
 # avoid sudo lecture during tests
 su -c 'echo foobar | sudo --stdin whoami' - admin
 
-# create user account for running the test
-if ! id runtest 2>/dev/null; then
-    useradd -c 'Test runner' runtest
-    # allow test to set up things on the machine
-    mkdir -p /root/.ssh
-    curl https://raw.githubusercontent.com/cockpit-project/bots/main/machine/identity.pub  >> /root/.ssh/authorized_keys
-    chmod 600 /root/.ssh/authorized_keys
-fi
-chown -R runtest "$SOURCE"
-
 # disable core dumps, we rather investigate them upstream where test VMs are accessible
 echo core > /proc/sys/kernel/core_pattern
 
-## CSR specific setup ##
-# install cockpit-packagekit and glibc-langpack-en for testAppMenu
-dnf install -y cockpit-packagekit glibc-langpack-en
-
-mkdir -p /var/log/journal/
-cp $FILES/1.journal /var/log/journal/1.journal
-cp $FILES/binary-rec.journal /var/log/journal/binary-rec.journal
-
-systemctl enable --now cockpit.socket
-
-# Run tests as unprivileged user
-# once we drop support for RHEL 8, use this:
-# runuser -u runtest --whitelist-environment=TEST_BROWSER,TEST_ALLOW_JOURNAL_MESSAGES,TEST_AUDIT_NO_SELINUX,SOURCE,LOGS $TESTS/run-test.sh
-runuser -u runtest --preserve-environment env USER=runtest HOME=$(getent passwd runtest | cut -f6 -d:) $TESTS/run-test.sh
-
-RC=$(cat $LOGS/exitcode)
-exit ${RC:-1}
+sh test/vm.install
+
+# Run tests in the cockpit tasks container, as unprivileged user
+CONTAINER="$(cat .cockpit-ci/container)"
+exec podman \
+    run \
+        --rm \
+        --shm-size=1024m \
+        --security-opt=label=disable \
+        --volume="${TMT_TEST_DATA}":/logs:rw,U --env=LOGS=/logs \
+        --volume="$(pwd)":/source:rw,U --env=SOURCE=/source \
+        --volume="${FILES}":/files:rw,U \
```

Comment:
I'm not sure what actually ends up using this, but I don't see any reference to `/files` being introduced anywhere alongside this commit, which seems wrong.  Maybe pair it with `--env=FILES=/files` (as is done for LOGS and SOURCE, above) to approximate the old behaviour?